### PR TITLE
[Gumtree.com] Re-enable ruleset and add subdomains

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by youdly** [comment]
- [Gumtree.co.za.xml](https://github.com/EFForg/https-everywhere/pull/14110)
- [Gumtree.com.au.xml](https://github.com/EFForg/https-everywhere/pull/14112)
- [Gumtree.ie.xml](https://github.com/EFForg/https-everywhere/pull/14111)
- [Gumtree.pl.xml](https://github.com/EFForg/https-everywhere/pull/14114)
- [Gumtree.sg.xml](https://github.com/EFForg/https-everywhere/pull/14113)

File: src/chrome/content/rules/Gumtree.com.xml [hunk 1]
```diff
@@ -0,0 +1,546 @@
...
+		- wwww	(m)
+		- www.wwwwww	(m)
+		- xmaswww	(m)
+		- xn--www-rp0a	(m)
+		- xr16www	(m)
+		- yearswww	(m)
+		- york	(m)
+		- yzfwww	(m)
+		- ztrwww	(m)
+		- zxwww	(m)
```
Comment (J0WI) [inline src/chrome/content/rules/Gumtree.com.xml]: There is a wildcard DNS record, so it makes no sense to list random subdomains here.

**Comment by J0WI** [comment]
`EBay.xml` contains a reference to the deleted `gumtree.com.xml`.

**Comment by youdly** [comment]
The reference has been removed in https://github.com/EFForg/https-everywhere/pull/13773

**Comment by J0WI** [comment]
So this depends on #13773 which has a huge list of dependencies itself.

**Comment by youdly** [comment]
yes, but after https://github.com/EFForg/https-everywhere/pull/13773#issuecomment-355730754, it currently depends on https://github.com/EFForg/https-everywhere/pull/13820 and https://github.com/EFForg/https-everywhere/pull/13879

**Comment by J0WI** [comment]
@youdly this needs more test urls for the wildcard.

**Comment by youdly** [comment]
Removed the wildcard

**Comment by J0WI** [comment]
> Removed the wildcard

Would you like to document it?

See also the errors exposed by Travis.

**Comment by youdly** [comment]
Added comment, removed problematic subdomain, rebased, squashed commits.

**Pull request merged**